### ISSUE TITLE
Code for round-trip accuracy?

### ISSUE THREAD

**Issue by najwalb** [kaist-amsg/LocalRetro#20]
The paper reports both ground-truth and round-trip accuracies. The repo does not provide any information about the round-trip accuracy however. Could you share the parameters you used in MolecularTransformer to get the numbers you reported in the paper? For example, what was your beam_size, n_best, etc?

**Comment by shuan4638** [comment]
Hi, thanks for interested in this project. I finished this project quite long time ago, but I think I simply used the default hyperparameters. I trained the Transformer by USPTO-50K dataset instead of USPTO-MIT dataset.

**Comment by shuan4638** [comment]
I will close this issue if there is no more discussion on this.

**Comment by najwalb** [comment]
Hi, thanks a lot for your reply, it's quite helpful :)
